src/BlazorWebView/src/Maui/PublicAPI/net6.0-windows10.0.19041/PublicAPI.Unshipped.txt

```diff
@@ -44,6 +44,8 @@ Microsoft.AspNetCore.Components.WebView.UrlLoadingStrategy.CancelLoad = 2 -> Mic
 Microsoft.AspNetCore.Components.WebView.UrlLoadingStrategy.OpenExternally = 0 -> Microsoft.AspNetCore.Components.WebView.UrlLoadingStrategy
 Microsoft.AspNetCore.Components.WebView.UrlLoadingStrategy.OpenInWebView = 1 -> Microsoft.AspNetCore.Components.WebView.UrlLoadingStrategy
 Microsoft.Extensions.DependencyInjection.BlazorWebViewServiceCollectionExtensions
+override Microsoft.AspNetCore.Components.WebView.Maui.BlazorWebView.OnPropertyChanged(string? propertyName = null) -> void
+override Microsoft.AspNetCore.Components.WebView.Maui.BlazorWebView.OnPropertyChanging(string? propertyName = null) -> void
```

Comment:
One thing I don't like is that the workaround technically involves changing the public API. Removing the workaround in the future when the root cause is fixed could eventually result in us changing the public API again (or changing the implementations of these two methods to call their respective base implementations). Thoughts?

Reply:
This is just an override, so I don't think it's breaking to add or remove it, no matter what.

Reply:
Well, what I was worried about is if someone were to derive from `BlazorWebView` and, for example, override `OnPropertyChanging` without calling the base implementation. Then since `Window_Destroying` doesn't ever get unsubscribed from the `Window.Destroying` event, we could end up invoking `WebView2.Close()` after the WebView control has already been removed. So relying on the customer to call the base implementation explicitly seemed a bit risky to me.

I suppose changing our requirements from "you must call the base implementation" to "ok now you don't have to" isn't technically breaking, as you pointed out. And it could be unlikely that anyone will ever hit this scenario.

Do you think this is a concern we should consider?

Reply:
> without calling the base implementation

Then I'd say that's a bug in whoever is deriving from this. We can of course always try to protect against every single thing, but given that in this case the pattern is to call the base implementation... which interestingly, I don't see us doing here 😁 So should we be calling base?

Reply:
Oh! I assumed the base implementation was empty. Which is ironically the assumption I was afraid the customer would make. Ok, so knowing that it would be a bug to not call the base implementation either way makes me a bit happier.